docs/ADR/0023-string-interpolation-and-binaries.md

```
@@ -0,0 +1,410 @@
+2 + 2 = 4
+> "{name length} chars"
+5 chars
+> "Braces: \{escaped\}"
+Braces: {escaped}
+> "No braces here"
+No braces here
+```
+
+### Error Examples
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)